### PR TITLE
libmo_unpack

### PR DESCRIPTION
**Pull request by ocefpaf**
There is a new version (3.1.2), but that needs new `build.sh`. I am being lazy here and sending the one I have.

### CODE REVIEW

**Comment by pelson** [comment]
It's missing the check dependency. I'm happy to pick this up in the next week or so.

It might also be worth adding @rhattersley as a maintainer if he is :+1:

File: recipes/libmo_unpack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+about:
+    home: https://github.com/SciTools/libmo_unpack
+    license: BSD 3-Clause
+    summary: Met Office PP/FieldsFile compression schemes
+
+extra:
+    recipe-maintainers:
+        - pelson
+        - rhattersley
```
Comment (ocefpaf) [inline recipes/libmo_unpack/meta.yaml]: @rhattersley I add you as a maintainer here and on the dependencies. Let me know if that is OK or if you want me to remove it from some (or all) the packages.

**Comment by pelson** [comment]
Looks like there were a couple of failures for OSX:

```
FAIL: ii-0041-test
FAIL: ii-0050-test
```

Ironically, I'd be happy enough if you didn't do a ```make check```. :fearful:

**Comment by ocefpaf** [comment]
> Looks like there were a couple of failures for OSX

I guess I kind of expected that. I lost count of how many `make check`s I had to by-pass on OS X. One day that will bite us back :worried:

File: recipes/libmo_unpack/meta.yaml (hunk 1 above)
Comment (rhattersley) [inline recipes/libmo_unpack/meta.yaml]: It's fine - thanks for checking.

**Pull request merged**